### PR TITLE
INTMDB-733: [CFN-PI] Add example for all constructors

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Proposed changes

_Jira ticket:_ [INTMDB-733](https://jira.mongodb.org/browse/INTMDB-733)

See [src/README.md](https://github.com/mongodb/awscdk-resources-mongodbatlas/blob/INTMDB-733/src/README.md)

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
  expected)
- [ ] This change requires a documentation update

## Required Checklist:

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation (if appropriate)
- [ ] I have run `make fmt` and formatted my code
- [ ] I have tested the CDK constructor in a CFN stack. See [TESTING.md](../TESTING.md)

## Next Steps:

- Open a follow-up PR to update examples with resources that use Stack properties

### CODE REVIEW

File: examples/l1-resources/alert-configuration.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+}
+
+export class CdkTestingStack extends cdk.Stack {
+  constructor(scope: Construct, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    const atlasProps = this.getContextProps();
+
+
+    const alert = new CfnAlertConfiguration(this, 'MyAlert', {
```
[the remark this answers is not in the document]
Comment (andreaangiolillo) [reply examples/l1-resources/alert-configuration.ts:20]: alert-configuration is a little different than the other resources: you have to provide `GroupId` in the `prop` of the stack. See [alert-configuration/index.ts](https://github.com/mongodb/awscdk-resources-mongodbatlas/blob/209df46360d9cadb5b8ddd5a4aa5d442eecec530/src/l1-resources/alert-configuration/index.ts#L1701). Keep in mind that right now when generating the CDK from the CFN we get several errors for the alert-configuration, there is a ticket to fix them. I will make update the ticket to fix also groupId.
I updated the example and added a comment highlighting that GroupId must be a property of the stack

[the remark this answers is not in the document]
Comment (maastha) [reply examples/l1-resources/alert-configuration.ts:20]: @andreaangiolillo As discussed, let's create a ticket for a follow-up on this to update these examples (including online-archive and others) with how to set GroupId (or other required properties) in the stack properties.

File: examples/l1-resources/online-archive.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+interface AtlasStackProps {
+  readonly projId: string;
+  readonly profile: string;
+  readonly collName: string;
+  readonly dbName: string;
+}
+
+export class CdkTestingStack extends cdk.Stack {
+  constructor(scope: Construct, id: string, props?: cdk.StackProps) {
+    super(scope, id, props); // Make sure that props contains "ProjectId".
```
Comment (maastha) [inline examples/l1-resources/online-archive.ts:15]: same comment applicable as in alert-configuration

**Review by maastha — APPROVED** [review]
Approving so we can address comments in a follow-up PR. Other than that LGTM

**Pull request merged**